2023-02-25 20:03:36,369 INFO    MainThread:22740 [wandb_setup.py:_flush():68] Configure stats pid to 22740
2023-02-25 20:03:36,369 INFO    MainThread:22740 [wandb_setup.py:_flush():68] Loading settings from C:\Users\anant\.config\wandb\settings
2023-02-25 20:03:36,369 INFO    MainThread:22740 [wandb_setup.py:_flush():68] Loading settings from C:\Users\anant\CS6910\Assignment 1\wandb\settings
2023-02-25 20:03:36,369 INFO    MainThread:22740 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-02-25 20:03:36,369 INFO    MainThread:22740 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-02-25 20:03:36,370 INFO    MainThread:22740 [wandb_init.py:_log_setup():492] Logging user logs to C:\Users\anant\CS6910\Assignment 1\wandb\run-20230225_200336-tjv3ziju\logs\debug.log
2023-02-25 20:03:36,370 INFO    MainThread:22740 [wandb_init.py:_log_setup():493] Logging internal logs to C:\Users\anant\CS6910\Assignment 1\wandb\run-20230225_200336-tjv3ziju\logs\debug-internal.log
2023-02-25 20:03:36,370 INFO    MainThread:22740 [wandb_init.py:_jupyter_setup():438] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001B6AED94610>
2023-02-25 20:03:36,372 INFO    MainThread:22740 [wandb_init.py:init():532] calling init triggers
2023-02-25 20:03:36,372 INFO    MainThread:22740 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-02-25 20:03:36,372 INFO    MainThread:22740 [wandb_init.py:init():588] starting backend
2023-02-25 20:03:36,372 INFO    MainThread:22740 [wandb_init.py:init():592] setting up manager
2023-02-25 20:03:36,378 INFO    MainThread:22740 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-02-25 20:03:36,385 INFO    MainThread:22740 [wandb_init.py:init():599] backend started and connected
2023-02-25 20:03:36,402 INFO    MainThread:22740 [wandb_run.py:_label_probe_notebook():1203] probe notebook
2023-02-25 20:03:36,404 INFO    MainThread:22740 [wandb_run.py:_label_probe_notebook():1213] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-02-25 20:03:36,404 INFO    MainThread:22740 [wandb_init.py:init():687] updated telemetry
2023-02-25 20:03:36,466 INFO    MainThread:22740 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2023-02-25 20:03:39,522 WARNING MsgRouterThr:22740 [router.py:message_loop():77] message_loop has been closed
2023-02-25 20:03:41,548 ERROR   MainThread:22740 [wandb_init.py:init():1160] error
Traceback (most recent call last):
  File "C:\Users\anant\anaconda3\lib\site-packages\wandb\sdk\wandb_init.py", line 1133, in init
    run = wi.init()
  File "C:\Users\anant\anaconda3\lib\site-packages\wandb\sdk\wandb_init.py", line 730, in init
    result = run_init_handle.wait(
  File "C:\Users\anant\anaconda3\lib\site-packages\wandb\sdk\lib\mailbox.py", line 271, in wait
    raise MailboxError("transport failed")
wandb.errors.MailboxError: transport failed
